pom.xml

```diff
@@ -34,10 +34,23 @@
     <name>Gravitee.io APIM - Resource - OAuth2 Provider API</name>
 
     <properties>
+        <gravitee-apim.version>4.9.0-alpha.2</gravitee-apim.version>
```

Comment:
it's an independent library, you should not import apim-bom if you don't want to end in circular dependencies.
remove this line